packages/datetime/src/dateInput.tsx

```diff
@@ -423,6 +425,15 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
         this.safeInvokeInputProp("onBlur", e);
     };
 
+    private handleInputKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+        if (e.which === Keys.ENTER) {
+            const nextValue = this.createMoment(this.state.valueString);
+            const nextDate = fromMomentToDate(nextValue);
+            this.handleDateChange(nextDate, true);
+            this.inputRef.blur();
```

Comment:
🤔 this `.blur()` feels intuitively wrong, as it kills the focus flow: I can't hit `enter` and then `tab` to move to the next form input.

can we do everything else but simply not blur? same goes for DRI.

Reply:
I buy that.